travis/compile.sh

```diff
@@ -74,6 +74,7 @@ $TS \
 --with-kerberos \
 --enable-sysvmsg \
 --enable-zend-test=shared \
+--enable-werror \
```

Comment:
appveyor equivalent required?

Reply:
Nope, the Windows build is nowhere near warning free.

Reply:
Yep, there are a lot of warnings like `warning C4267: "=": Konvertierung von "size_t" nach "uint32_t", Datenverlust möglich` and similar. Many of these issues would likely affect Linux as well. :(

Reply:
I would recommend to suppress the integer truncation and signed/unsigned mismatch warnings in the Windows build system, so that other more important warnings become visible. Right now it's hard to spot a new warning showing up among all the noise.

Once everything else is fixed, we can consider enabling them again.